### DISCUSSION TITLE
Editor Customizations

### DISCUSSION THREAD

**Discussion by gfeeng** [greatfrontend/greatfrontend#5] [Ideas/Feature Requests]
A list of things we want to allow customization:

- Key bindings: Vim/Emacs
- Syntax highlighting: Code syntax highlighting theme is like a developer's clothing, it forms a developer's identity and is an important part of how they present themselves. GFE's code blocks and editor uses a non-changeable Dracula theme and we should allow users to customize code syntax highlighting to their favorite themes if possible.
- Spaces/Tabs
- Tab Size
- TypeScript
- Disable syntax highlighting
- Disable autocompletion

Leave a comment on your favorite editor theme you'd like to see supported!

**Comment by arthur326** [comment]
Vim key bindings would be great!
